core/heap.c

```diff
@@ -112,18 +112,17 @@ static const uint BLOCK_SIZES[] = {
     /* 40 dbg / 36 rel: */
     ALIGN_FORWARD(sizeof(fragment_t) + sizeof(indirect_linkstub_t), HEAP_ALIGNMENT),
 #if defined(X64)
-    sizeof(instr_t), /* 104 x64 */
 #    ifdef DEBUG
     sizeof(fragment_t) + sizeof(direct_linkstub_t) +
         sizeof(cbr_fallthrough_linkstub_t), /* 112 dbg x64 / 104 rel x64 */
 #    else
-/* release == instr_t */
+    sizeof(instr_t), /* 112 x64 */
```

Comment:
Do we now have one element less for DEBUG ?

Also the position of `sizeof(instr_t)` is different in BLOCK_SIZES now. Does that matter? I don't see any other adjustment to BLOCK_SIZES reads.

Reply:
No.  There are asserts that alert on any invariant violation in the size sequence.